packages/twenty-front/src/modules/object-record/record-filter/states/context/RecordFiltersComponentInstanceContext.ts

```diff
@@ -0,0 +1,4 @@
+import { createComponentInstanceContext } from '@/ui/utilities/state/component-state/utils/createComponentInstanceContext';
```

Comment:
I'm just re-thinking about this. I'm puzzled between the granular approach we have to introduce:
- ContextStoreInstanceContext
- RecordFilterInstanceContext
- RecordSortInstanceContext

i wonder if everything should'nt be part of the ContextStore to ease the experience. We could keep the RecordFilter tooling as it is but loading everything from a ContextStoreWrapper (ContextStoreInstanceContext /RecordFilterInstanceContext / RecordSortInstanceContext) and make them share a same instanceId.

Not sure about it, open to discussion. I'm also fine with the current implementation.
Also fine with putting everything as part of the ContextStoreInstanceContext and remove RecordFilterInstanceContext